README.md

```diff
@@ -50,32 +51,33 @@ See the online documentation: [Getting started with Event Store Cloud](https://d
 
 ## Client libraries
 
-This getting started guide shows you how to get started with EventStoreDB by setting up an instance or cluster and configuring it.
+This guide shows you how to get started with EventStoreDB by setting up an instance or cluster and configuring it.
 EventStoreDB supports two protocols: gRPC and TCP(legacy).
 
-EventStoreDB supported clients
+EventStoreDB supported gRPC clients
+
 - Go: [EventStore/EventStore-Client-Go](https://github.com/EventStore/EventStore-Client-Go)
 - .NET: [EventStore/EventStore-Client-Dotnet](https://github.com/EventStore/EventStore-Client-Dotnet)
 - Java: [(EventStore/EventStoreDB-Client-Java](https://github.com/EventStore/EventStoreDB-Client-Java)
 - Node.js (javascript/typescript): [EventStore/EventStore-Client-NodeJS](https://github.com/EventStore/EventStore-Client-NodeJS)
 - Rust: [EventStore/EventStoreDB-Client-Rust](https://github.com/EventStore/EventStoreDB-Client-Rust)
 - Read more in the [gRPC clients documentation](https://developers.eventstore.com/clients/grpc)
 
-Legacy TCP Clients
-- .Net: [EventStoreDB-Client-Dotnet-Legacy](https://github.com/EventStore/EventStoreDB-Client-Dotnet-Legacy)
-- JVM: [EventStore.JVM](https://github.com/EventStore/EventStore.JVM)
-- Haskell: [EventStoreDB-Client-Haskell](https://github.com/EventStore/EventStoreDB-Client-Haskell)
+Community supported gRPC clients
 
-Community developed clients
 - Elixir: [NFIBrokerage/spear](https://github.com/NFIBrokerage/spear)
 - Python: (coming soon)
```

Comment:
https://pypi.org/project/esdbclient/

Reply:
@ylorph a comment and a question: 
- We were planning to add all of the python info in another PR since we want to add it in multiple READMEs and code examples.
- Is there a strong need to link to pypi instead of the repo? The README navigation does not work on pypi, and the content is exactly the same on the repo.

Reply:
Excellent. Thanks.